client/src/components/Template/util.ts

```diff
@@ -0,0 +1,36 @@
+import { unified } from 'unified';
+import remarkParse from 'remark-parse';
+import remarkGfm from 'remark-gfm';
+import remarkRehype from 'remark-rehype';
+import rehypeStringify from 'rehype-stringify';
+import rehypeSanitize, {defaultSchema} from 'rehype-sanitize';
+
+import Handlebars from 'handlebars';
+
+const processor = unified()
+  .use(remarkParse)
+  .use(remarkGfm)
+  .use(remarkRehype, {allowDangerousHtml: true})
```

Comment:
Why `allowDangerousHtml`?

Reply:
That was left-over from another experiment, I've removed it